cltk/corpus/greek/tlg/parse_tlg_indices.py

```diff
@@ -125,68 +131,65 @@ def get_date_author():
 
 def get_dates():
     """Return a list of all the epithet labels."""
-    _path = os.path.join(THIS_DIR, 'author_date.json')
-    _dict = open_json(_path)
+    _dict = get_date_author()
     return sorted(_dict.keys())
 
 
 def get_date_of_author(_id):
     """Pass author id and return the name of its associated date."""
-    _path = os.path.join(THIS_DIR, 'author_date.json')
-
-    _id = str(_id)
-
-    _dict = open_json(_path)
-
+    _dict = get_date_author()
     for date, ids in _dict.items():
         if _id in ids:
             return date
+    return None
 
 
 def _get_epoch(_str):
     """Take incoming string, return its epoch."""
-    if _str.startswith('B.C. '):
-        return 'bc'
-    elif _str.startswith('A.D. '):
-        return 'ad'
-    elif _str.startswith('a. B.C. '):
-        return 'bc'
-    elif _str.startswith('p. B.C. '):
-        return None  #?
+    _return = None
+    if _str.startswith('A.D. '):
+        _return = 'ad'
     elif _str.startswith('a. A.D. '):
-        return None  #?
+        _return = None #?
     elif _str.startswith('p. A.D. '):
-        return 'ad'
-    else:
-        return None
+        _return = 'ad'
+    elif regex.match(r'^[0-9]+ B\.C\. *', _str):
+        _return = 'bc'
+    elif regex.match(r'^a\. *[0-9]+ B\.C\. *', _str):
+        _return = 'bc'
+    elif regex.match(r'^p\. *[0-9]+ B\.C\. *', _str):
+        _return = None  #?
+    elif _str == 'Incertum' or _str == 'Varia':
+        _return = _str
+    return _return
+
+
+def _check_number(_str):
+    """check if the string contains only a number followed by ?"""
+    if regex.match(r'^[0-9]+\?*', _str):
+        return True
+    return False
 
 
 def _handle_splits(_str):
     """Check if incoming date has a '-" or '/', if so do stuff."""
-
+    _str = _str.replace('/', '-')
```

Comment:
Even though they have different meanings, we need to split them with the same rules. so I just replaced <code>'/'</code> with <code>'-'</code> so that we can handle them directly with a single case : <code>-</code>.